static/app/components/events/interfaces/spans/spanDetail.tsx

```diff
@@ -421,20 +420,23 @@ function SpanDetail(props: Props) {
                       )}
                     >
                       Span ID
-                      <Clipboard
-                        value={`${window.location.href.replace(
-                          window.location.hash,
-                          ''
-                        )}#span-${span.span_id}`}
-                      >
-                        <StyledIconLink />
-                      </Clipboard>
                     </SpanIdTitle>
                   )
                 }
                 extra={renderTraversalButton()}
               >
-                {span.span_id}
+                <Fragment>
```

Comment:
Is Fragment needed since it's a child of Row? Maybe just wrong typing in Row?

Reply:
ah yeah, typing could be better instead of Fragment. i'll fix that.